src/utils/globals.ts

```diff
@@ -1,3 +1,8 @@
+import { SessionIdManager } from '../sessionid'
+import { ErrorEventArgs, ErrorProperties, Properties } from '../types'
+import { PostHog } from '../posthog-core'
+import { SurveyManager } from '../extensions/surveys'
```

Comment:
Errrr surely this isn't what you want...

Reply:
Although I am surprised the bundle didn't grow much it still seems like we should never be importing from extensions...

Reply:
Ah... the fun of offsite brain tiredness. Should have a linter maybe to save in future... 🤔

Reply:
nicer if we had interfaces instead of these `any`s but alreayd a lot changing in here so 🙈

Reply:
Technically we could do `import type` and that would give you what you want but thats also at risk of getting changed and importing the actual data too...